test/require.cjs

```diff
@@ -0,0 +1,24 @@
+// a test to see if CJS require still works
+
+const { Aedes } = require('../aedes.js')
+const defaultExport = require('../aedes.js')
+const { test } = require('tap')
+
+test('test Aedes constructor', function (t) {
```

Comment:
was going to ask for this, well done :) Don't know if this still requires the experimental flag btw?